cmd/utils/flags.go

```diff
@@ -750,6 +750,10 @@ var (
 		Usage: "External EVM configuration (default = built-in interpreter)",
 		Value: "",
 	}
+	CheckStaticCallCacheFlag = cli.BoolFlag{
+		Name:  "vm.checkcache",
+		Usage: "If true, this checks the results of the static call cache. Note: this flag is experimental and will be removed",
```

Comment:
I might recommend "temporary" instead of "experimental", although I am not actually sure why we would remove this flag in the future. It seems like something helpful to have for debugging should we adjust this code in the future and as we add new calls that use it.

Reply:
I'll switch it to temporary. Maybe "developer" is another option. I think we may keep this for a couple releases, but I want to be able to remove it in the future.